### DISCUSSION TITLE
Unable to build appimage

### DISCUSSION THREAD

**Discussion by zer09** [tauri-apps/tauri#4683] [Q&A] [answered]
I tried to the `npm create tauri-app` to create a sample project, but unable to build it.

First I generate the project using `npm create tauri-app` command and choose svelte framework.
Then execute `npm install` followed by `npm run tauri build` to confirm it will run.
But after executing `npm run tauri build` I get this
```
 Finished release [optimized] target(s) in 4m 23s
    Bundling poss_0.1.0_amd64.deb (/home/gc/tauri/poss/src-tauri/target/release/bundle/deb/poss_0.1.0_amd64.deb)
    Bundling poss_0.1.0_amd64.AppImage (/home/gc/tauri/poss/src-tauri/target/release/bundle/appimage/poss_0.1.0_amd64.AppImage)
       Error failed to bundle project: error running appimage.sh: error running appimage.sh: `failed to run /home/gc/tauri/poss/src-tauri/target/release/bundle/appimage/build_appimage.sh`
```
Got an error on building the appimage.

here is my tauri info
```
✗ npm run tauri info

> poss@1.0.0 tauri
> tauri "info"


Environment
  › OS: Manjaro 21.3.3 X64
  › Node.js: 14.17.2
  › npm: 8.9.0
  › pnpm: 6.11.0
  › yarn: 1.22.17
  › rustup: 1.25.1
  › rustc: 1.62.0
  › cargo: 1.62.0
  › Rust toolchain: stable-x86_64-unknown-linux-gnu 

Packages
  › @tauri-apps/cli [NPM]: 1.0.4
  › @tauri-apps/api [NPM]: 1.0.2
  › tauri [RUST]: 1.0.4,
  › tauri-build [RUST]: 1.0.4,
  › tao [RUST]: 0.12.1,
  › wry [RUST]: 0.19.0,

App
  › build-type: bundle
  › CSP: unset
  › distDir: ../public
  › devPath: http://localhost:8080/
  › framework: Svelte
  › bundler: Rollup

App directory structure
  ├─ src
  ├─ public
  ├─ node_modules
  ├─ src-tauri
  └─ .vscode
```

**Comment by erayerdin** [comment, marked as answer]
I had this error the other day. I observed this sometimes happens due to conflicting files on each build. There are two actions you can take:

# 1st Method: Faster but Prone to Further Error

You simply delete the build files for AppImage by doing:

```bash
rm src-tauri/target/release/bundle/appimage* -rfv
```

And then you run `npm run tauri build` or `cargo tauri build` again so it will start building AppImage from start. Since it won't compile the app from scratch, this is a faster method. However, you might still get a different error with this.

# 2nd Method: Slower but More Solid

You simply delete your whole build directory by doing:

```bash
rm src-tauri/target/ -rfv
```

This, however, will recompile your app from the start, so it is much slower.

***

So, pick your poison.

**Reply by zer09** [reply]
I already tried method 2, but still got same error. When I tried to build with `--verbose` I got this error `Could not find dependency: libffi.so.6` when I try to syslink libffi I a got this `Could not find dependency: libcroco-0.6.so.3`

**Reply by FabianLars** [reply]
Do you have VMware installed? Weird question, but see https://github.com/tauri-apps/tauri/issues/4163 for more.

**Reply by zer09** [reply]
Yes I have VMware installed, I will check #4163

**Comment by Raduc4** [comment]
Has anyone solved this error on Arch Linux?
`ERROR: Could not find dependency: libicui18n.so.72`
```
pacman -Qi icu
Name            : icu
Version         : 73.2-2
```

**Reply by EndorFlem** [reply]
Same error

**Reply by sean-clayton** [reply]
I'm pretty sure we need `icu` @ Version 72. arch :c
